src/pages/tasks/NewTaskDetailsPage.js

```diff
@@ -109,6 +109,7 @@ function NewTaskDetailsPage(props) {
                         accessibilityLabel={props.translate('task.title')}
                         value={taskTitle}
                         onValueChange={(value) => setTaskTitle(value)}
+                        autoCorrect={false}
```

Comment:
@jasperhuangg I couldn't understand from your last comment here if we're disabling for all the platforms? I think we shouldn't.

cc - @dukenv0307

Reply:
quick bump @jasperhuangg